docs/guides/cloud/flaky-test-management.mdx

```diff
@@ -56,6 +56,8 @@ tests by Cypress Cloud. Therefore, **enabling
 [test retries](/guides/guides/test-retries#Configure-Test-Retries) is required**
 to take advantage of any flaky test management feature.
 
+If you want more control over the pass/fail result of any test deemed flaky by test retries, you may want to use [experimental test retries](/guides/references/experiments#Experimental-Retries), which is available as of Cypress `13.x.x`.
+
```

Comment:
This might be a preview link/user error issue, but this links me to the experiment table instead of the "Experimental Test Retries (available as of Cypress 13.x.x)" section

Reply:
oops, good catch!